heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
+    "access": "external",
+    "image": {
+        "name": "heapoverflow",
+        "tags": [
+            "latest"
+        ]
+    },
+    "logo": "",
+    "info": "",
+    "depends": [],
+    "config": [],
+    "ports": [],
+    "volumeMounts": [],
+    "additionalResources": [],
+    "repositories": [],
+    "command": [],
```

The default command in this container is `bash`, which will terminates without a tty attached - this won't work in the cluster because we need the container to stay running. But `bash` is great for running locally, so we can override this command only for our cluster by changing the `command` here.

This will override the default command in the container and instead run a command that never terminates to keep the container alive:
```suggestion
    "command": [ "/bin/sh", "-c", "while true; do sleep 30; done;" ],
```

There is currently a bug in the API prevents this `command` from being used at runtime, but I'm working on fixing this issue soon in a separate PR: https://github.com/nds-org/workbench-apiserver-python/pull/13

When accessing the terminal through the UI, the `exec` command will use bash (and falls back to a couple others), so the user experience should remain the same when using it in CHEESEHub :+1: